src/main/java/com/group6/accommodation/domain/reservation/repository/ReservationRepository.java

```diff
@@ -14,7 +16,17 @@
 
 public interface ReservationRepository extends JpaRepository<ReservationEntity, Long> {
 
-    
+
+    default ReservationEntity getById(Long id) {
```

Comment:
```suggestion
    default ReservationEntity getByIdOrElseThrow(Long id) {
```

로 바꾸면 가독성이 좋아질 것 같은데 어떻게 생각하시나요??

Reply:
밑에 있는 에외를 보니 병준님이 말씀하신게 더 좋아보이네요.